src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/Reissuance/ReissuanceConsent/HealthCertificateReissuanceConsentViewModel.swift

```diff
@@ -96,15 +112,16 @@ final class HealthCertificateReissuanceConsentViewModel {
 						)
 					]
 				)
-			]
-		)
+			)
+		}
 	}
 
 	func markCertificateReissuanceAsSeen() {
 		certifiedPerson.isNewCertificateReissuance = false
 	}
 
 	func submit(completion: @escaping (Result<Void, HealthCertificateReissuanceError>) -> Void) {
+		/*
```

Comment:
is fixed with other PR: https://github.com/corona-warn-app/cwa-app-ios/pull/4485